### ISSUE TITLE
KeyError: 'type'

### ISSUE THREAD

**Issue by jdbranham** [sparkmeter/sentry2csv#10]
Thanks for this app!

I have an interesting error when running -  
I'm not a python developer, but it looks like the 'type' field may not be available.  
Any thoughts?  

```
Fetching issues page 1
Fetching issues page 2
Fetching issues page 3
Traceback (most recent call last):
  File "/usr/local/bin/sentry2csv", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/sentry2csv/sentry2csv.py", line 133, in main
    loop.run_until_complete(export(args.token[0], args.organization[0], args.project[0], enrich=enrichments))
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.7/site-packages/sentry2csv/sentry2csv.py", line 107, in export
    write_csv(outfile, issues)
  File "/usr/local/lib/python3.7/site-packages/sentry2csv/sentry2csv.py", line 83, in write_csv
    "Error": issue["metadata"]["type"],
KeyError: 'type'
```

**Comment by arusahni** [comment]
Hi! Sorry for the delayed response - I made a note to get back to this, and then life intervened.

Could you please share the commands you invoked the script with? Feel free to redact as needed, I just need to know the shape of the command.

**Comment by jdbranham** [comment]
No worries @arusahni - life does that to me all the time =)

I execute it like this - `sentry2csv --token xxxxx orgName projectName` 
Maybe there's something unique in the project config? 
I'm not sure - I just joined the project for some temp assistance, and I've left it. I might still have access to the sentry site for a bit, if you'd like me to look at anything.

**Comment by arusahni** [comment]
I've just released a new version (v1.1.1) with some improved error handling and debug output.  If you're still able to query the Sentry project, could you give it another try? You'll most likely have to use the `-vv` flags to get some actual error logs.

Once you do, can you share the line containing: "Failed to process row, missing key" as well as the line after? The latter is the entire record that failed to be written to a CSV.  Feel free to redact values - the schema/shape of the record is all that's important to me.

Thanks for helping me work through this!

**Comment by arusahni** [comment]
I believe this was fixed by #14 (reported by another user in #13).
